### PR TITLE
DDT-1085: Define TOs

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: jit-transfer-obj/pom.xml [hunk 1]
```diff
@@ -24,6 +24,18 @@
...
 		</dependency>
+		<dependency>
+			<groupId>org.dcsa.shared-kernel</groupId>
+			<artifactId>dcsa-shared-kernel-domain</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.dcsa.shared-kernel</groupId>
+			<artifactId>dcsa-shared-kernel-infrastructure</artifactId>
+		</dependency>
+		<dependency>
```
Comment (FinnAtAsseco) [inline jit-transfer-obj/pom.xml:35]: TO should not reference persistence

**Review by FinnAtAsseco — CHANGES_REQUESTED** [review]

File: jit-transfer-obj/pom.xml (hunk 1 above)
Comment (mp-assco) [reply jit-transfer-obj/pom.xml]: Removed

File: jit-transfer-obj/pom.xml [hunk 2]
```diff
@@ -24,6 +24,14 @@
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-annotations</artifactId>
 		</dependency>
+		<dependency>
```
Comment (FinnAtAsseco) [inline jit-transfer-obj/pom.xml:27]: Hmm, TO's really shouldn't depend on anything in shared-kernel actually. What do you need in domain and infrastructure that causes you to put these dependencies?

Comment (mp-assco) [reply jit-transfer-obj/pom.xml:27]: ValidVesselIMONumber and FacilityTypeCode/Address.

Comment (FinnAtAsseco) [reply jit-transfer-obj/pom.xml:27]: Dammit, yeah.

File: jit-transfer-obj/src/main/java/org/dcsa/jit/enums/ModeOfTransport.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package org.dcsa.jit.enums;
```
Comment (FinnAtAsseco) [inline jit-transfer-obj/src/main/java/org/dcsa/jit/enums/ModeOfTransport.java:1]: package should be org.dcsa.jit.transferobjects.enums

Comment (mp-assco) [reply jit-transfer-obj/src/main/java/org/dcsa/jit/enums/ModeOfTransport.java:1]: Ah, right. Moved!

File: jit-transfer-obj/pom.xml (hunk 2 above)
Comment (mp-assco) [reply jit-transfer-obj/pom.xml:27]: Removed infrastructure dependency.

**Review by nt-as — APPROVED** [review]

**Review by FinnAtAsseco — APPROVED** [review]

**Pull request merged**